app/uk/gov/hmrc/transitmovementsrouter/models/MessageTypes.scala

```diff
@@ -16,6 +16,12 @@
 
 package uk.gov.hmrc.transitmovementsrouter.models
 
+import play.api.libs.json.JsError
+import play.api.libs.json.JsString
+import play.api.libs.json.JsSuccess
+import play.api.libs.json.Reads
+import play.api.libs.json.Writes
+
 sealed trait MessageType extends Product with Serializable {
   def code: String
   def rootNode: String
```

Comment:
Don't we need associated auditType and movementType ?

Reply:
I think you mean MessageTypes -  this is an existing class, not sure we can update as part of this ticket

Reply:
I'm in two minds about this, because really, the audit types themselves weren't part of this ticket, just creating the connector and service was. Another ticket was about firing the message type audit events and I'd have expected this type association to be done then -- but I'd have also expected the `AuditType`s to have been added in another ticket too.

Let's make this ticket about the service/connector only, and make sure it matches what the API service/connectors look like. The associations can be done in the ticket for firing message type messages.